sources/api/apiserver/src/server/mod.rs

```diff
@@ -36,6 +36,7 @@ fn notify_unix_socket_ready() -> Result<()> {
         ensure!(
             Command::new("systemd-notify")
                 .arg("--ready")
+                .arg("--no-block")
```

Comment:
Do we need to worry about any of the service units defined with `Type=notify` having the issue you ran into that required this, or does systemd handle it OK internally?  I don't have enough context.

Reply:
The out-of-tree ones would need to be updated to use the new barrier function, and that doesn't seem to have happened yet.

I expect we'd hit the same SELinux denials in that case. I'm OK with changing the policy if we do.

Reply:
By out-of-tree you mean not shipped with systemd/dbus?  I think that's all of our `Type=notify` cases except for dbus-broker itself, so 8 service units.  I'm assuming you didn't see SELinux denials, though, so I may be misunderstanding.

Reply:
By out-of-tree I meant not part of systemd:
* apiserver
* containerd
* dbus-broker
* wickedd

In-tree ones could have been updated to use the barrier function. I doubt the others would have since the systemd release is so new and they would need a release cycle to adopt it, and also need a reason to worry about the race condition the feature was meant to address.

`apiserver` is unusual in that it uses the `systemd-notify` client directly, which now defaults to blocking via the barrier function.